app/components/footer.tsx

```diff
@@ -110,25 +111,40 @@ function AboutSection() {
         Full time educator teaching people development
       </p>
 
-      <div className="text-secondary flex mt-6 space-x-4">
-        <a
-          className="hover:text-primary focus:text-primary focus:outline-none"
-          href={externalLinks.github}
-        >
-          <GithubIcon />
-        </a>
-        <a
-          className="hover:text-primary focus:text-primary focus:outline-none"
-          href={externalLinks.youtube}
-        >
-          <YoutubeIcon />
-        </a>
-        <a
-          className="hover:text-primary focus:text-primary focus:outline-none"
-          href={externalLinks.twitter}
-        >
-          <TwitterIcon />
-        </a>
+      <div className="text-secondary flex items-center justify-between mt-6 md:flex-col md:items-start">
+        <div className="flex space-x-4">
+          <a
+            className="hover:text-primary focus:text-primary focus:outline-none"
+            href={externalLinks.github}
+          >
+            <GithubIcon />
+          </a>
+          <a
+            className="hover:text-primary focus:text-primary focus:outline-none"
+            href={externalLinks.youtube}
+          >
+            <YoutubeIcon />
+          </a>
+          <a
+            className="hover:text-primary focus:text-primary focus:outline-none"
+            href={externalLinks.twitter}
+          >
+            <TwitterIcon />
+          </a>
+        </div>
+
+        <div className="relative flex items-center w-24 md:mt-20 lg:w-32">
+          <img
+            className="absolute dark:block hidden w-full opacity-50"
+            src={images.kentSignatureDarkMode()}
+            alt={images.kentSignatureDarkMode.alt}
+          />
+          <img
+            className="absolute block dark:hidden w-full opacity-50"
+            src={images.kentSignatureLightMode()}
+            alt={images.kentSignatureLightMode.alt}
+          />
```

Comment:
If we had the image in a gray shade, close to `gray-500` or `blueGray-500`, we could replace these two images with a single one.